include/lemlib/devices/gyro/imu.hpp

```diff
@@ -55,23 +55,23 @@ class Imu : public Gyro {
          *
          * @return float heading, in radians, locked from -pi to pi
```

Comment:
```suggestion
         * @return Angle heading
```
Update doxygen comment